Source/Maths/CoordinateTransformations.hpp

```diff
@@ -44,25 +48,29 @@ static Tensor<2, data_t> spherical_jacobian(const data_t x, const double y,
     return jac;
 }
 
-// Incerse Jacobian
-template <class data_t>
-static Tensor<2, data_t>
-inverse_spherical_jacobian(const data_t x, const double y, const double z)
+// Inverse Jacobian
+static Tensor<2, amrex::Real>
+inverse_spherical_jacobian(const amrex::Real x, const double y, const double z)
 {
     // calculate useful position quantities
-    data_t rho2 = simd_max(x * x + y * y, 1e-12);
-    data_t rho  = sqrt(rho2);
-    // NOLINTNEXTLINE(readability-identifier-length)
-    data_t r2 = simd_max(x * x + y * y + z * z, 1e-12);
-    data_t r  = sqrt(r2);
+    // NOLINTBEGIN [readability-identifier-length]
+    amrex::Real rho2     = x * x + y * y;
+    amrex::Real min_rho2 = 1e-12;
+    rho2                 = (rho2 < min_rho2) ? min_rho2 : rho2;
+    amrex::Real rho      = std::sqrt(rho2);
+    amrex::Real min_r2   = 1e-12;
+    amrex::Real r2       = x * x + y * y + z * z;
+    r2                   = (r2 < min_r2) ? min_r2 : r2;
+    amrex::Real r        = std::sqrt(r2);
```

Comment:
For consistency with the previous function:
```suggestion
    amrex::Real r2      = x * x + y * y + z * z;
    amrex::Real r       = std::sqrt(r2);
    amrex::Real min_r   = 1e-6;
    r                   = std::max(r, min_r);
```

Reply:
As noted above.